strapi/strapi
# Add missing swedish core admin translations

### What does it do?

Added missing swedish translations for the core/admin part of strapi

### Why is it needed?

The swedish translations were mostly unfinished, so using the swedish language for the admin interface was mostly in english.

### How to test it?

Build the application and set the admin interface language to "sv", and check so that everything is correctly translated
